src/app/sort/sort.component.spec.ts

```diff
@@ -62,7 +62,6 @@ describe('Sort component - ', () => {
 
     let button = element.querySelector('.sort-pf button');
     button.click();
-    fixture.detectChanges(); // Workaround to fix dropdown tests
     tick();
```

Comment:
Can you please add the "Workaround to fix dropdown tests" comment. When folks cut/paste tests, I want to be clear as to why this is being used.

Reply:
No problem, I didn't know if we were still considering it a workaround :) I'll add this comment back.